bcipy/signal/model/gaussian_mixture/gaussian_mixture.py

```diff
@@ -12,43 +12,16 @@
 warnings.filterwarnings("ignore")  # ignore DeprecationWarnings from tensorflow
 
 
-class KernelGP(SignalModel):
-    def __init__(self):
-        reshaper = GazeReshaper()
-
-    def fit(self, training_data: np.ndarray, training_labels: np.ndarray):
-        training_data = np.asarray(training_data)
-
-    def evaluate(self, test_data: np.ndarray, test_labels: np.ndarray):
-        ...
+class GaussianProcess(SignalModel):
```

Comment:
This is missing several important methods. It doesn't seem like it could be used online as-is